processor/src/test/java/org/mapstruct/ap/test/bugs/_3485/Issue3485Test.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._3485;
+
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.compilation.annotation.CompilationResult;
+import org.mapstruct.ap.testutil.compilation.annotation.Diagnostic;
+import org.mapstruct.ap.testutil.compilation.annotation.ExpectedCompilationOutcome;
+
+/**
+ * @author hduelme
+ */
+@IssueKey("3463")
```

Comment:
```suggestion
@IssueKey("3485")
```

Seems to be copied from `Issue3463Test` class.

Reply:
@KENNYSOFT thanks for reviewing this. Seems like I forgot to change that

Reply:
Resolved in next PR